doc/intf/olo_intf_i2c_master.md

```diff
@@ -57,6 +57,7 @@ tristate buffers.
 | :----------------- | :------ | ------- | :----------------------------------------------------------- |
 | ClkFrequency_g     | real    | -       | Frequency of the clock _Clk_ in Hz.<br />For correct operation, the clock frequency must be at least 16x higher than the I2C bus frequency. |
 | I2cFrequency_g     | real    | 100.0e3 | Frequency of the _I2cScl_ signal in Hz                       |
```

Comment:
I think this one deserves a comment that this is the SCL frequency without prescaling.